crates/sui-framework/packages/dwallet-system/sources/proof.move

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) dWallet Labs, Ltd.
+// SPDX-License-Identifier: BSD-3-Clause-Clear
+
+// TODO (#228): Remove this module.
+/// The proof module
+/// Responsible to start and manage the Proof generation MPC flow
+/// Used only for testing the way we launch & manage an MPC flow.
+module dwallet_system::proof {
```

Comment:
Is there a reason why this is called a "proof" module? The MPC protocol generates signatures so the naming seems odd to me.

Reply:
Until I finish implementing the MPC traits for the 2pc-mpc, they are using a simple `Maurer` asynchrnous statement aggregation protocol to "test" the paper you wrote and they are implementing.